index.d.ts

```diff
@@ -1113,7 +1113,15 @@ declare namespace WAWebJS {
         clearState: () => Promise<boolean>,
         /** Deletes the chat */
         delete: () => Promise<boolean>,
-        /** Loads chat messages, sorted from earliest to latest. */
+        /**
+         * Loads chat messages, sorted from earliest to latest.
+         * @param {Object} searchOptions Options for searching messages. Right now only limit is supported.
+         * @param {Number} [searchOptions.limit] The amount of messages to return. If no limit is specified, the available messages will be returned. Note that the actual number of returned messages may be smaller if there aren't enough messages in the conversation. Set this to Infinity to load all messages.
+         * @param {Number} [searchOptions.minTimestamp] If specified, filter message by minimum timestamp.
```

Comment:
Probably minimum can be rephrased to oldest and maximum earliest

Reply:
also, millis or unix

Reply:
i personally prefer "minimum timestamp" but I can change it...